test/e2e/daemon_set.go

```diff
@@ -61,8 +63,120 @@ var _ = Describe("Daemon set", func() {
 		f.afterEach()
 	})
 
-	It("should launch a daemon pod on every node of the cluster", func() {
-		testDaemonSets(f)
+	It("should run and stop simple daemon", func() {
+		ns := f.Namespace.Name
+		c := f.Client
```

Comment:
And I think that simpleDsName and complexDsName can collapse into simply dsName, the assignment of which can also move to BeforeEach(). The two tests will run in different namespaces, so there will be no name clash.